tarea 2/Luis Ernesto/index.html

```diff
@@ -0,0 +1,20 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Alertas Personalizadas</title>
+    <script>
+        let tamaño = 0;
+        let result = prompt  ("Digita un numero");
+        tamaño = result;
```

Comment:
puedes simplificar esto a una línea:
```
        const result = prompt("Digita un numero");
```

si quieres usar 0 como valor por defecto (si el usuario no introduce nada, por ejemplo), puedes usar:

```suggestion
        const result = prompt("Digita un numero") || 0;
```